src/main/java/org/opensearch/ad/ml/CheckpointDao.java

```diff
@@ -100,6 +116,7 @@
     public static final String ENTITY_RCF = "rcf";
     public static final String ENTITY_THRESHOLD = "th";
     public static final String FIELD_MODEL = "model";
+    public static final String FIELD_MODELV2 = "modelV2";
```

Comment:
Do you want to make it a combination of the version string in RCF 2.0? So that if it becomes 2.1-rcX instead of 2.0-rc2 then you would not have to rename anything. That being said, the renaming can also be done then.

Reply:
This is for checkpoint format.  As long as rcf checkpoint format does not change, we don't have to touch this part, even if we move to rcX.